### DISCUSSION TITLE
Welcome to Open Notebook Discussions — Start Here

### DISCUSSION THREAD

**Discussion by lfnovo** [lfnovo/open-notebook#1] [Announcements]
Open Notebook is built in the open, and we want contribution to begin before the code.

Discussions are where the community can help us understand problems, challenge trade-offs, explore possible directions, and validate whether changes deliver what they promised.

## Choose the right place

📣 **[Announcements](https://github.com/lfnovo/open-notebook/discussions/categories/announcements)**  
Releases, project updates, decisions, and outcomes shared by the maintainers.

🧭 **[Feedback Requests](https://github.com/lfnovo/open-notebook/discussions/categories/feedback-requests)**  
Focused questions and Current Bets opened by the maintainers. Bring your experience, constraints, counterexamples, and concerns. These are invitations to help shape our thinking—not votes.

💡 **[Ideas](https://github.com/lfnovo/open-notebook/discussions/categories/ideas)**  
Problems, needs, and possible directions proposed by the community. Start with what you were trying to accomplish, what made it difficult, who it affects, and how you handle it today. A proposed solution is welcome, but not required.

🙏 **[Q&A](https://github.com/lfnovo/open-notebook/discussions/categories/q-a)**  
Questions about using, configuring, or understanding Open Notebook.

💬 **[General](https://github.com/lfnovo/open-notebook/discussions/categories/general)**  
Community conversations that do not fit the categories above.

## Discussions or Issues?

Use a **Discussion** while we are still trying to understand whether something should be done, which problem matters, or what direction makes sense.

Use an **[Issue](https://github.com/lfnovo/open-notebook/issues/new/choose)** for:

- a reproducible bug or diagnosable installation failure;
- work the project has explicitly accepted;
- a change with a clear expected outcome and enough context to execute.

A popular idea is not automatically a commitment. Maintainers remain responsible for deciding what the project will support and sustain.

## How work moves forward

This is the direction we are building toward, one small process change at a time:

1. A signal or problem enters a Discussion.
2. The community adds use cases, constraints, alternatives, and counterexamples.
3. A maintainer records the decision and what remains open.
4. If the work is ready, an Issue is created with the relevant context and expected outcome.
5. When the work ships, the result and learning return to the original Discussion.

Anyone can propose. Many people can help shape and challenge the direction. Maintainers own the commitments.

## Help us keep the conversation useful

- Search for an existing Discussion before starting a new one.
- Add concrete context whenever possible.
- Challenge ideas, not people.
- Do not begin a large implementation until a maintainer confirms that the work is ready.
- Follow our [Code of Conduct](https://github.com/lfnovo/open-notebook/blob/main/CODE_OF_CONDUCT.md).

Thank you for helping us build not only the code, but a better way to contribute to Open Notebook.

**Comment by caduzex13** [comment]


**Reply by lfnovo** [reply]
Please write a separate issue about it. Not sure what you mean?
